package.json

```diff
@@ -23,8 +23,10 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "nestjs-yup": "^1.0.13",
```

Comment:
What the package is it? Attach the link please.

Reply:
I planned to use it with yup for validation. But now I found another solution. I created validationPipe for Yup, so I will remove this package.

Reply:
Please remove and don`t use different unpopular libraries, they can be unstable.